backend/logic/order/index.js

```diff
@@ -194,6 +194,30 @@ async function processNewOrder({
   const order = await Order.create(orderObj)
   log.info(`Saved order ${order.fqId} to DB.`)
 
+  if (shopConfig.inventory) {
+    const cartItems = get(data, 'items', [])
+    const dbProducts = await Product.findOne({
+      shopId: shop.id,
+      productId: cartItems.map((item) => item.product)
```

Comment:
I did not realize findOne can take a list of productIds... and there is not need for a where clause?

Reply:
Sorry, I'm might have messed up this, will fix this